cypress/features/regression/experimental/textboxValidations.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Textbox validations component
+    I want to change Textbox validations component properties
+
+    Background: Open Textbox validations component page
+        Given I open "Experimental Textbox" component page validations
+
+    @positive 
+    @validations
+    Scenario Outline: Verify the <state> validation of Textbox component
+        Given Type "<keyWord>" text into input
+            And I click outside of the component
+        When I hover mouse onto icon
+        Then tooltipPreview on preview is set to '<text>'
+            And icon on preview is "<state>"
+        Examples:
+            | state   | keyWord | text                                            |
+            | info    | info    | This value should be longer than 12 characters  |
```

Comment:
why we should use the same values in `state` and `keyWord` columns?

Reply:
fixed